### ISSUE TITLE
Can't get elements to pin?

### ISSUE THREAD

**Issue by MandyMadeThis** [janpaepke/ScrollMagic#311]
I can't figure out what I'm doing wrong and I'd really appreciate some help. 

Here is my code: http://codepen.io/MandyMadeThis/pen/PqPrQd?editors=001

This is my desired effect:http://sticky-element.com/

I'd like the tab's (tab 1, tab 2, etc) to pin to the bottom of the header and then be 'pushed' out of the way when the following tab hits the bottom of the previous tab.

I cannot seem to get any of the tabs to pin at all...any help would be greatly appreciated.

Also, I'd like the triggerHook to be the height of the fixed header - right now I've just approximated that but if there is a better way, please let me know?

Thank you!

**Comment by janpaepke** [comment]
Hi Mandy,

first off: Never use minified versions for development.
Not because it makes tracing errors more difficult in the code, but also because logging is often disabled or limited.

If you change your pen to use `ScrollMagic.js` instead of `ScrollMagic.min.js` the console starts spitting out some valuable insights as to what is going wrong.

The first mistake is that you use variables to store the target elements as jQuery objects.
ScrollMagic doesn't natively support them, unless you add the [jQuery plugin for ScrollMagic](http://scrollmagic.io/docs/framework.jQuery.html).
Instead you could just supply selectors to ScrollMagic.
But so that I need to change less code, let's include the jQuery plugin.

Next you need to better understand what `triggerHook` and `offset` mean and how they play together.
I recommend this tut: https://ihatetomatoes.net/svg-scrolling-animation-triggered-scrollmagic/

In short:
The offset is always relative to the `triggerElement` and the `triggerHook` is related to scroll container size.

So if your hook is the top of the page and the pins should be offset by the height of the header it seems obvious that you have to set the `triggerHook` to `0` or `"onLeave"` and the `offset` to the height of the header. For all scenes.

Once you do that things start to look better: http://codepen.io/janpaepke/pen/pJEoga

I'm guessing this is your desired outcome?

**Comment by MandyMadeThis** [comment]
AH! Thank you so much. I'm a new developer and obviously still learning, so I really appreciate you taking the time to point out the obvious (to you).
